docs/client-libraries-producers.md

```diff
@@ -78,37 +101,22 @@ This example shows how to send messages using producers.
 
    ```go
     msg := pulsar.ProducerMessage{
-        Payload: []byte("Here is some message data"),
-        Key: "message-key",
-        Properties: map[string]string{
-            "foo": "bar",
-        },
-        EventTime: time.Now(),
-        ReplicationClusters: []string{"cluster1", "cluster3"},
+        Payload: []byte("my-sync-message"),
     }
 
     if _, err := producer.send(msg); err != nil {
       log.Fatalf("Could not publish message due to: %v", err)
     }
    ```
 
-    For all methods of the `ProducerMessage` object, see [here](https://pkg.go.dev/github.com/apache/pulsar-client-go/pulsar#ProducerMessage).
+  For all methods of the `ProducerMessage` object, see [here](https://pkg.go.dev/github.com/apache/pulsar-client-go/pulsar#ProducerMessage).
```

Comment:
```suggestion
  For all methods of the `ProducerMessage` object, see [Go API doc](https://pkg.go.dev/github.com/apache/pulsar-client-go/pulsar#ProducerMessage).
```
Show the doc name explicitly instead of "here" for screen readers' convenience.